### PR TITLE
fheroes2: Adding additional installation instructions via caveats

### PR DESCRIPTION
**Pull request by kfishster**
Similar to #120685 but this time doing it via caveats rather than puts/ohai.

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/fheroes2.rb [hunk 1]
```diff
@@ -43,6 +43,34 @@ def install
...
+
+      As an alternative to the previous step, you can manually copy the subdirectories 'ANIM', 'DATA', 'MAPS' and 'MUSIC' (some of them may be missing depending on the version of the original game) from the original game directory to the '.fheroes2' subdirectory of your home directory. When all prerequisites are in place, you can start the game by typing:
+
+      fheroes2
+
+      Please visit project website:
+
+      #{homepage}
+
+      and read #{prefix}/share/doc/fheroes2/README.txt for more information.
```
Comment (SMillerDev) [inline Formula/fheroes2.rb:70]: Only this seems specific to Homebrew, the rest has to be in upstream documentation

Comment (kfishster) [reply Formula/fheroes2.rb]: Removed instructions, left only links to site and doc location

File: Formula/fheroes2.rb [hunk 2]
```diff
@@ -43,6 +43,20 @@ def install
...
+    <<~EOS
+      Please visit project website:
+
+      > #{homepage}
+
+      and read
+
+      > #{prefix}/share/doc/fheroes2/README.txt
+
+      for full installation instructions and more information.
```
Comment (SMillerDev) [inline Formula/fheroes2.rb:56]: People already rarely read caveats so this won't help. I've reduced it to the only part that is specific to Homebrew.
```suggestion
      Documentation is available at:
      #{prefix}/share/doc/fheroes2/README.txt
```

File: Formula/fheroes2.rb [hunk 3]
```diff
@@ -43,6 +43,13 @@ def install
     bin.install "script/homm2/extract_homm2_resources.sh" => "fheroes2-extract-resources"
   end
 
+  def caveats
+    <<~EOS
+      Documentation is available at:
+      #{prefix}/share/doc/fheroes2/README.txt
```
Comment (SMillerDev) [inline Formula/fheroes2.rb:49]:
```suggestion
      #{share}/doc/fheroes2/README.txt
```

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**